src/vm/win32threadpool.cpp

```diff
@@ -392,11 +393,11 @@ BOOL ThreadpoolMgr::Initialize()
 
     // initialize Worker and CP thread settings
     DWORD forceMin;
-    forceMin = CLRConfig::GetConfigValue(CLRConfig::INTERNAL_ThreadPool_ForceMinWorkerThreads);
+    forceMin = Configuration::GetKnobDWORDValue(W("System.Threading.ThreadPool.MinThreads"), CLRConfig::INTERNAL_ThreadPool_ForceMinWorkerThreads);
```

Comment:
Could you please factor this out into a small helper method, so that the actual Configuration call is done in just single place?